pkgs/by-name/cg/cgproxy/package.nix

```
@@ -0,0 +1,48 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, libbpf
+, nlohmann_json
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "cgproxy";
+  version = "0.20";
+
+  src = fetchFromGitHub {
+    owner = "springzfx";
+    repo = finalAttrs.pname;
+    hash = "sha256-mI57YGB0wG2ePHb8HXFth6g7QXrz8x5gGv2or0oBrEA=";
+    rev = "v${finalAttrs.version}";
```

```suggestion
    repo = "cgproxy";
    rev = "v${finalAttrs.version}";
    hash = "sha256-mI57YGB0wG2ePHb8HXFth6g7QXrz8x5gGv2or0oBrEA=";
```
repo will always be this, no matter what pname is.